2025-03-30 11:11:43,594 - ERROR - Error initializing MFS directory: HTTPConnectionPool(host='127.0.0.1', port=5001): Max retries exceeded with url: /api/v0/files/mkdir?arg=%2Fpatents&parents=true (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000025852E20B50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2025-03-30 11:11:58,136 - INFO - Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
2025-03-30 11:12:00,825 - ERROR - Error in save_and_upload: HTTPConnectionPool(host='127.0.0.1', port=5001): Max retries exceeded with url: /api/v0/add (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000025856ED2E90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2025-03-30 11:12:11,176 - INFO - Successfully verified IPFS hash: QmbixBStjKNeqFA7DKtnAfA5bDUNDtYbUwL5mWCcdksic8
